inspectit-ocelot-documentation/docs/getting-started/docker-examples.md

```diff
@@ -30,6 +30,16 @@ Uses Prometheus Server for metrics gathering and storage, Grafana for Dashboards
 
 ![Demo scenario using Prometheus and Jaeger](assets/demo-landscape-prometheus-jaeger.png)
 
+### Demo #3 - Wavefront and Zipkin
+
+Sends Zipkin traces to Wavefront through a Wavefront proxy running as a Docker container.
+Note: Before running the demo, you must set the Wavefront host and API token as follows:
```

Comment:
Can you move this line into a blockquote then its highlighted and easier to notice. 
So, doing it like this:

```
Sends Zipkin traces to Wavefront through a Wavefront proxy running as a Docker container.

* File: `docker-compose-wavefront-zipkin.yml`

> Note: Before running the demo, you must set the Wavefront host and API token as follows:
> ```
> export WAVEFRONT_URL=https://<your cluster>.wavefront.com/api
> export WAVEFRONT_TOKEN=<API token obtained from Wavefront>
> ```
```